### ISSUE TITLE
Lost previous contributions at 50k

### ISSUE THREAD

**Issue by mcomella** [mozilla/MozStumbler#1831]
From bugzilla https://bugzilla.mozilla.org/show_bug.cgi?id=1239377:

Steps to reproduce:

Reports are lost after reaching 50,000 reports 

Actual results:

All the stumbled data was lost 

Expected results:

The data should not be lost

**Comment by Nidhiya** [comment]
Any idea if there was a maximum limit to the identifier. As per my observation, it got reset once it reached 50,000.

**Comment by pordini** [comment]
This likely happened to me. Last I checked it was around 40.000 ready to send, now it's ~3000.
No log entries about it, nothing. 

Just another frustrating experience. Time to uninstall and forget about this app.

**Comment by JonnyTech** [comment]
Yes, it happens to me all the time, so many contributions disappear regularly from the app when I am unable to get online...
